README.md

```diff
@@ -1,20 +1,22 @@
-
 # groundhog
+`groundhog` is a service and client libraries to access NASA's publicly available surface elevation data, [SRTM](https://www2.jpl.nasa.gov/srtm/).  Given a table with GPS information, this package provides utilities to enrich that dataset with elevation and slope features.  Just load the container and one of the clients, and you're all set! 
```

Comment:
could you put a blank line between headings and their body content?

Reply:
added